core/src/main/java/org/mskcc/cbio/oncokb/Constants.java

```diff
@@ -21,7 +21,7 @@ public final class Constants {
     public static final String FIVE_UTR = "5_prime_UTR_variant";
     public static final String UPSTREAM_GENE = "upstream_gene_variant";
 
-    public static final String PUBLIC_API_VERSION = "v1.4.0";
+    public static final String PUBLIC_API_VERSION = "v1.5.0";
     public static final String PRIVATE_API_VERSION = "v1.4.1";
```

Comment:
Technically if you add endpoint, it could just be patch. We upgrade to minor if API changes are significant(parameter update, deletion etc). But here I'd like to get users' attention about this new endpoint, to emphasize the importance of the endpoint. @calvinlu3